include/tcg/tcg-op-gvec.h

```diff
@@ -218,6 +218,27 @@ typedef struct {
     bool write_aofs;
 } GVecGen4;
 
+typedef struct {
+    /*
+     * Expand inline as a 64-bit or 32-bit integer. Only one of these will be
+     * non-NULL.
+     */
+    void (*fni8)(TCGv_i64, TCGv_i64, TCGv_i64, TCGv_i64, int64_t);
+    void (*fni4)(TCGv_i32, TCGv_i32, TCGv_i32, TCGv_i32, int32_t);
+    /* Expand inline with a host vector type.  */
+    void (*fniv)(unsigned, TCGv_vec, TCGv_vec, TCGv_vec, TCGv_vec, int64_t);
+    /* Expand out-of-line helper w/descriptor, data in descriptor.  */
+    gen_helper_gvec_4 *fno;
+    /* The optional opcodes, if any, utilized by .fniv.  */
+    const TCGOpcode *opt_opc;
+    /* The vector element size, if applicable.  */
+    uint8_t vece;
+    /* Prefer i64 to v64.  */
+    bool prefer_i64;
+    /* Write aofs as a 2nd dest operand.  */
+    bool write_aofs;
```

Comment:
``write_aofs`` doesnt' seem to be needed right now. Do you foresee it being needed soon? Or should we leave it out until we actually have a need for it?

Reply:
I can remove it if you prefer, I've kept it mostly for consistency with `GVecGen4`.

Reply:
I would remove it, but it's no big deal either way. Regarding consistency, one could also argue that we should replace write_aofs with load_dest (after all, you mention in the commit message that the implementation of tcg_gen_gvec_4i follows tcg_gen_gvec_3i (which uses load_dest, and not write_aofs).

Reply:
`GVecGen3` has a `load_dest` field, so `GVecGen3i` copied that. `GVecGen4` has `write_aofs`, so I kept that in `GVecGen4i`. I guess I'll drop it anyway, it can be added in the future as needed.

Reply:
Fixed in https://github.com/PPC64/qemu/pull/35/commits/18bc72f49226ba5607c1133457d086dfc7846ac9